crates/primitives/src/compression/mod.rs

```diff
@@ -1,6 +1,7 @@
+#[cfg(feature = "std")]
+use std::thread_local;
```

Comment:
please use full paths instead of feature gating imports

Reply:
Remove macro above import?

Reply:
make sense

Reply:
done